espflash/src/cli/monitor/mod.rs

```diff
@@ -28,20 +28,18 @@ use strum::{Display, EnumIter, EnumString, VariantNames};
 
 use crate::{
     cli::{
-        monitor::{
-            parser::{InputParser, ResolvingPrinter},
-            symbols::Symbols,
-        },
+        monitor::parser::{InputParser, ResolvingPrinter},
         MonitorConfigArgs,
     },
     connection::{reset::reset_after_flash, Port},
+    image_format::Metadata,
 };
 
 pub mod external_processors;
 pub mod parser;
 
 mod line_endings;
-mod symbols;
+pub(crate) mod symbols;
```

Comment:
Sorry if I've missed something, but why does the visibility of this module need to change?

Reply:
It shouldn't I just forgot to undo it, thanks